docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -490,6 +490,8 @@ A simulant's adherence score **does NOT change** during the simulation and will
 The below table shows the percent chance of being assigned different buckets of adherence. Adherence is 
 randomly assigned to all simulants. 
 
+Adherence is only tracked and observed for those prescribed a medication.
```

Comment:
I'd remove the word "tracked". Technically the way it's been implemented we will be "tracking" adherence for everybody - we just won't observe unless they're prescribed a medication.

Reply:
Limited to just observed - thanks